verification/negativeCases.go

```diff
@@ -0,0 +1,275 @@
+// Licensed under the Apache-2.0 license
+
+package verification
+
+import (
+	"errors"
+	"testing"
+)
+
+var InvalidHandle = ContextHandle{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15}
+
+func TestInvalidHandle(d TestDPEInstance, c DPEClient, t *testing.T) {
+	testInvalidHandle(d, c, t)
+}
+
+// Checks whether error is reported when non-existent handle is passed as input to DPE commands.
+// Exceptions are - GetProfile, InitializeContext, GetCertificateChain, GetTaggedTCI commands
+// which do not need context handle as input parameter.
+func testInvalidHandle(d TestDPEInstance, c DPEClient, t *testing.T) {
```

Comment:
All this code can just be in the public function instead of calling a separate helper (same for all of these tests)

Reply:
Removed the helper, put the logic under public function